test/nightly/global.nightly.test.ts

```diff
@@ -0,0 +1,54 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ResourceManagementClient } from 'azure-arm-resource';
+import { WebSiteManagementClient } from 'azure-arm-website';
+import { IHookCallbackContext } from 'mocha';
+import * as vscode from 'vscode';
+import { TestAzureAccount } from 'vscode-azureextensiondev';
+import { AzExtTreeDataProvider, AzureAccountTreeItemWithProjects, createAzureClient, ext } from '../../extension.bundle';
+import { longRunningTestsEnabled } from '../global.test';
+
+export let testAccount: TestAzureAccount;
+export let testClient: WebSiteManagementClient;
+export const resourceGroupsToDelete: string[] = [];
+
+// Runs before all nightly tests
+suiteSetup(async function (this: IHookCallbackContext): Promise<void> {
```

Comment:
Do test files prefixed with global guarantee that it'll be run first?

Reply:
I don't think so. Honestly I don't entirely remember all the rules on this. Mostly I was copying the comment/pattern from `global.test.ts` [here](https://github.com/microsoft/vscode-azurefunctions/blob/master/test/global.test.ts#L21)

I _believe_ the key part is `suiteSetup` outside of `suite` (which is how this file is set up) will run before any `suiteSetup` inside a `suite` (which is how most other test files are set up).